src/FloatingActionButton/FloatingActionButton.js

```diff
@@ -270,7 +270,8 @@ class FloatingActionButton extends Component {
       style: Object.assign({},
         styles.icon,
         mini && styles.iconWhenMini,
-        iconStyle),
+        iconStyle,
+        this.props.children.props.style),
```

Comment:
I paid special attention on this.
For my logic (used here):
- user can fully control MUI version
- user will be able to fully control style of children inside FAB

Thus, user can:

1. override MUI style
2. follow MUI style, bu append new options

The only disadvantage that if user ovveride MUI style, he should test changes in new version of MUI to work properly with.

But if user-defined ```this.props.children.props.style``` is first, the user cannot to override MUI style, but only append to it. Using this solution will limit the functionality, but remove the disadvantage mentioned above.

Hence, I choose more flexibility.